src/iterative_sorting/iterative_sorting.py

```diff
@@ -1,30 +1,54 @@
-# TO-DO: Complete the selection_sort() function below 
+# Selection_sort() function 
+# Selection sort repeatedly finds the smallest element 
+
 def selection_sort( arr ):
-    # loop through n-1 elements
+    # Loop through n-1 elements
     for i in range(0, len(arr) - 1):
-        cur_index = i
-        smallest_index = cur_index
-        # TO-DO: find next smallest element
-        # (hint, can do in 3 loc) 
-             
-
-
+        
```

Comment:
Nice, super clean code. I believe the repo suggested naming some variables, clearly not required but would make the code easier to read if you hadn't left comments.